backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -17,11 +17,13 @@
 #include "client/crash_report_database.h"
 #include "client/settings.h"
 
+#if __ANDROID_API__ >= 23
```

Comment:
That's a lot of ifdef. Can we instead compile a file conditionally for API >= 23 (failing that, we could `#include` that file)?